code/WorkInProgress/telescope/quantumTelescope.dm

```diff
@@ -120,6 +120,7 @@ TODO: Enforce ping rate limit here as well in case someone futzes with the javas
 							var/disty = abs(vY - E.loc_y)
 							var/dist = (distx * distx + disty * disty) ** 0.5
 							if (dist <= E.size)
+								using.playsound_local(src.loc, "sound/machines/found.ogg", 50, 1)
```

Comment:
do you mean src.loc here

Reply:
When copying the line over from another place, I gave a short thought to the fact that I could change it to be src, seeing as the sound is played only for that person.
But then I thought that the sound is still coming from the console, so it kind of makes sense like this?

I can change it if you think it makes more sense to have it originate from the player though.